animate/interpolation.py

```diff
@@ -367,7 +367,10 @@ def clement_interpolant(source, target_space=None, boundary=False):
         instructions = {
             0: "t[i] = t[i] + v[0] * s[0]",
             1: "t[i, j] = t[i, j] + v[0] * s[0, j]",
-            2: f"t[i, {dim} * j + k] = t[i, {dim} * j + k] + v[0] * s[0, {dim} * j + k]",
+            2: (
+                f"t[i, {dim} * j + k] = "
+                f"t[i, {dim} * j + k] + v[0] * s[0, {dim} * j + k]"
```

Comment:
You always put the space at the beginning of the second line and I think you like to be consistent :)
```suggestion
                f"t[i, {dim} * j + k] ="
                f" t[i, {dim} * j + k] + v[0] * s[0, {dim} * j + k]"
```